app/constants/account-codes/ar-admin.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   SOS267: 'SOS267',
```

Comment:
I'd say that's not relevant.  They're just numbers chosen by Finance and I'm sure in their context they have some rationale for those.  I think it would just be admin burden to put some random text.